skimage/transform/radon_transform.py

```diff
@@ -216,26 +216,37 @@ def iradon(radon_image, theta=None, output_size=None,
     img = np.pad(radon_image, pad_width, mode='constant', constant_values=0)
 
     # Construct the Fourier filter
-    f = fftfreq(projection_size_padded).reshape(-1, 1)   # digital frequency
-    omega = 2 * np.pi * f                                # angular frequency
-    fourier_filter = 2 * np.abs(f)                       # ramp filter
+    n1 = np.arange(0, projection_size_padded / 2 + 1, dtype=np.int)
```

Comment:
Lets use floor divide here
```suggestion
    n1 = np.arange(0, projection_size_padded // 2 + 1, dtype=np.int)
```

Reply:
Seems like my suggestion doesn't give the same results
![image](https://user-images.githubusercontent.com/90008/53691027-eb391280-3d43-11e9-8f4a-b2d86410f1d6.png)

Reply:
projection_size_padded will always be even.

Reply:
Alright, good point, I would still prefer if we releaved our intentions on making this an int. but whatever here.

Reply:
Sure, actually we can remove the `dtype=np.int`. They are going to be promoted to floats later anyway. It was an unnecessary conversion on my part.

Reply:
Yeah, I just realized that this is such a small point and there are 1 million ways to compute the same thing, just leave it.

I guess I got confused looking at this again after so long since there were 2 key, and very important changes here.